internal/parser/scanner/rule_based_scanner_test.go

```diff
@@ -163,6 +184,7 @@ func _TestRuleBasedScannerWithRuleset(input string, ruleset ruleset.Ruleset, wan
 				break
 			}
 		}
+		fmt.Println(got)
```

Comment:
```suggestion
		t.Log(got)
```

Reply:
Was testing again, is it necessary to log it?